Realm/RLMRealm.mm

```diff
@@ -172,24 +176,16 @@ void RLMRealmAddPathSettingsToConfiguration(RLMRealmConfiguration *configuration
 }
 
 @implementation RLMRealm {
-    // Used for read-write realms
-    NSHashTable *_notificationHandlers;
     NSHashTable *_collectionEnumerators;
-
-    std::unique_ptr<ClientHistory> _history;
-    std::unique_ptr<SharedGroup> _sharedGroup;
-
-    // Used for read-only realms
-    std::unique_ptr<Group> _readGroup;
-
-    // Used for both
-    Group *_group;
-    BOOL _readOnly;
-    BOOL _inMemory;
-
-    NSData *_encryptionKey;
+    NSHashTable *_notificationHandlers;
 }
 
+@dynamic path;
```

Comment:
no need to mark as `@dynamic` since the getter for read-write properties is overriden